package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@example/design-system",
+  "name": "@nl-design-system-unstable/design-system",
```

Comment:
Is it logical this main package is called 'design system' I would expect this to be something like 'basis' or 'utilities'?